packages/devtools-launchpad/package.json

```diff
@@ -14,6 +14,7 @@
   "scripts": {
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
+    "license-check": "../devtools-license-check/bin/devtools-license-check",
```

Comment:
we should be able to replace this with. 

```
 "license-check": "devtools-license-check"
```

The only setup needed is that each component has to have `devtools-license-check` as a devDependency. Unfortunately, you need to make a `0.0.1` release to get this working the first time but after that it's actually symlinked by lerna so you can make changes to `devtools-license-check` and test it another package.